pkg/monitor/monitor/template.go

```diff
@@ -75,75 +72,30 @@ var (
 )
 
 func init() {
-	var err error
-	truePattern, err = config.NewRegexp("true")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	allMatchPattern, err = config.NewRegexp("(.+)")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	portPattern, err = config.NewRegexp("([^:]+)(?::\\d+)?;(\\d+)")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	tikvPattern, err = config.NewRegexp("tikv")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	pdPattern, err = config.NewRegexp("pd")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	tidbPattern, err = config.NewRegexp("tidb")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	addressPattern, err = config.NewRegexp("(.+);(.+);(.+);(.+)")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	tiflashPattern, err = config.NewRegexp("tiflash")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	pumpPattern, err = config.NewRegexp("pump")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	drainerPattern, err = config.NewRegexp("drainer")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	cdcPattern, err = config.NewRegexp("ticdc")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	importerPattern, err = config.NewRegexp("importer")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	lightningPattern, err = config.NewRegexp("tidb-lightning")
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	dmWorkerPattern, err = config.NewRegexp(dmWorker)
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
-	dmMasterPattern, err = config.NewRegexp(dmMaster)
-	if err != nil {
-		klog.Fatalf("monitor regex template parse error,%v", err)
-	}
+	truePattern = "true"
```

Comment:
let's move all the string variable initialization from init() function to the variable declaration block above.